src/features/Status/StatusCTAContainer.tsx

```diff
@@ -57,30 +79,94 @@ class StatusCTAContainer extends React.Component<IStatusCTAContainerProps, IStat
     }
   };
 
+  public modifyHacker = async (hacker: IHacker) => {
+    const newHacker = { ...hacker };
+    const { timeZone, country, city } = this.state;
+    newHacker.application.location = {
+      timeZone,
+      country,
+      city,
+    };
+    return newHacker;
+  };
+
   public render() {
+    const { timeZone, country, city } = this.state;
     return (
-      <Flex flex={1} flexDirection="column" justifyContent="center" alignItems="center" p="6px 30px 96px 30px">
+      <Flex
+        flex={1}
+        flexDirection="column"
+        justifyContent="center"
+        alignItems="center"
+        p="6px 30px 96px 30px"
+      >
         {this.props.confirmed && this.props.account ? (
           <StatusCTA
             status={this.state.status}
             firstName={this.props.account.firstName}
             settings={this.state.settings}
-            onClickConfirm={this.confirmStatus}
+            onClickConfirm={async () =>
+              this.setState({ isLocationModalOpen: true })
+            }
             // tslint:disable-next-line: jsx-no-lambda
             onClickWithdraw={() => this.setState({ isModalOpen: true })}
           />
         ) : (
           <ConfirmationEmailSentComponent />
         )}
+        <ConfirmModal
+          isOpen={this.state.isLocationModalOpen}
+          onCanceled={() => this.setState({ isLocationModalOpen: false })}
+          onConfirmed={this.confirmStatus}
+          cancelLabel="Cancel"
+          confirmLabel="Submit"
+        >
+          <Box mb={'10px'} alignSelf={'center'}>
```

Comment:
Can you add a title of "Confirm you spot" and a subtitle of "Please let us know where you'll be hacking from"? (or some text along those lines)